### PR TITLE
Event routes, handlers, and documentation.

### PR DESCRIPTION
**Pull request by evscott**
#67 #68 #69 

This pull request contains several changes:

1. Routes and handlers have been added for transportation, accommodation, and plans.
2. Verbose URL's have been added to routes.
3. Arguments are now passed through the body, and never the params.

### CODE REVIEW

File: server/controller/tripRoutes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+const middleware = require('../middleware');
+const tripHandler = require('../handlers/tripHandler');
+
+// All the routes for trips
+router.get('/get', middleware.checkToken, tripHandler.getTrips);
+router.post('/add', middleware.checkToken, tripHandler.addTrip);
+router.put('/update/:tid', middleware.checkToken, tripHandler.updateTrip);
```
Comment (gzinck) [inline server/controller/tripRoutes.js]: Should we remove the :tid here?

File: server/controller/triproutes.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-const middleware = require('../handlers/middleware');
+const middleware = require('../middleware');
 const tripHandler = require('../handlers/tripHandler');
 
 // All the routes for trips
-router.put('/:tid', middleware.checkToken, tripHandler.updateTrip);
-router.delete('/:tid', middleware.checkToken, tripHandler.deleteTrip);
-router.get('/', middleware.checkToken, tripHandler.getTrips);
-router.post('/', middleware.checkToken, tripHandler.addTrip);
+router.get('/get', middleware.checkToken, tripHandler.getTrips);
```
Comment (gzinck) [inline server/controller/triproutes.js]: This file is no longer used. Should it be deleted?

File: server/controller/tripRoutes.js (hunk 1 above)
Comment (evscott) [reply server/controller/tripRoutes.js]: Yes. Good catch.

File: server/controller/triproutes.js (hunk 2 above)
Comment (evscott) [reply server/controller/triproutes.js]: Which file? The 'triproutes' file? I've removed our general 'approutes' file.

Comment (gzinck) [reply server/controller/triproutes.js]: Yes: there is both a triproutes and a tripRoutes file right now (we want the latter, I assume)

Comment (evscott) [reply server/controller/triproutes.js]: Bizarre. It wasn't showing up on my local branch, but it was on the remote one. It's now been removed.

File: server/controller/tripRoutes.js (hunk 1 above)
Comment (gzinck) [reply server/controller/tripRoutes.js]: We should probably take the :tid out before we merge!

Comment (evscott) [reply server/controller/tripRoutes.js]: Whoops, yes!

**Pull request merged**